spaceward/src/routes/root.tsx

```diff
@@ -33,14 +33,14 @@ export default function Root() {
 		{},
 		10
 	);
-	const spacecount =
-		spacesQuery?.pages.length ||
-		(0 > 0 && spacesQuery?.pages[0].spaces?.length) ||
-		0;
+	const spacecount = spacesQuery?.pages[0].spaces?.length || 0;
 
 	const client = useClient();
 	const sendMsgNewSpace = client.WardenWarden.tx.sendMsgNewSpace;
 
+	console.log("spacesQuery", spacesQuery);
+	console.log("spacecount", spacecount);
+
```

Comment:
just checking, do we want to keep those?

Reply:
good spot, just removed those, thanks!